src/core/README.md

```diff
@@ -12,6 +12,7 @@
   * [MEME Suite] =4.9.0
   * [twoBitToFa]\*
   * [variants2matrix] (See below.)
+  * [pre-built expression matrix for cancers]
```

Comment:
This is not a hard dependency. Precalculated expression matrices are discussed under the [References](https://github.com/stjude/cis-x/tree/fc8639c5bf1ab127583c18714f3150a6966bd945/src/core#references) section of the README.

Reply:
Oh, that is my bad. we can leave it as it was.

Reply:
should I just change the README.md back and re-push it ?

Reply:
Yes, you can edit and repush anything in your `add-a-user-guide-link` branch.

Reply:
I change it back, should I pull another request?

Reply:
No, changes are updated in this PR: https://github.com/stjude/cis-x/pull/2/commits/5e42db2e3b334fba3d92c9fad59bc95977c3b988